### PR TITLE
Calculate TypeParameterKind based on the container type

### PR DESCRIPTION
**Pull request by nbalakin**

### CODE REVIEW

File: src/Compilers/VisualBasic/Portable/Symbols/Wrapped/WrappedTypeParameterSymbol.vb [hunk 1]
```diff
@@ -88,6 +88,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Public Sub New(underlyingTypeParameter As TypeParameterSymbol)
             Debug.Assert(underlyingTypeParameter IsNot Nothing)
             Me._underlyingTypeParameter = underlyingTypeParameter
+
```
Comment (nbalakin) [inline src/Compilers/VisualBasic/Portable/Symbols/Wrapped/WrappedTypeParameterSymbol.vb:91]: This whole class seems to be unused so I am unable to validate the assert.

**Review by AlekseyTs — APPROVED** [review]
LGTM (commit 2)

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler For the second review on a small community PR.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler For the second review on a small community PR.

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler For the second review on a small community PR.

**Review by cston — APPROVED** [review]

**Comment by AlekseyTs** [comment]
@nbalakin Thanks for the contribution.

**Pull request merged**